tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
```

Comment:
One other thing I should have told you about - we run a Travis CI job with every patch in this repo. That's covered by this file:
https://github.com/noironetworks/python-opflex-agent/blob/master/.travis.yml

We should add python3 tests to the Travis CI job as part of this patch.